test/cp.js

```diff
@@ -451,6 +451,19 @@ test('-R implies -P', t => {
   });
 });
 
+test('-Ru no longer ignores -u', t => {
+  const [source, dest] = ['foo', 'bar'].map(s => `test/resources/cp/cp-Ru/${s}/file`);
+  // First, update the dest file
+  shell.touch(dest);
+  // Get the old mtime for dest
+  const oldTime = fs.statSync(dest).mtimeMs;
+  // Now, copy the old dir to the new one
+  shell.cp('-Ru', source, dest);
+  // Get the new mtime for dest
+  const newTime = fs.statSync(dest).mtimeMs;
+  t.is(oldTime, newTime);
+});
+
```

Comment:
This test case is mostly lgtm, but you need another test case where src is newer than dest. `-u` means the copy should not happen. You should compare file contents to verify the copy does not occur (a successful copy would change file contents, but `-u` prevents the copy).

Reply:
If src is newer than dest, then the copy should (regardless of `-u`) occur. That said, checking file contents is a better way to test cp than checking timestamps, resolving an issue you raised in an earlier comment. I'll set it up to do that instead, and add the additional test.